src/main/java/com/phoenixnap/oss/ramlplugin/raml2code/interpreters/PojoBuilder.java

```diff
@@ -243,6 +244,8 @@ public PojoBuilder withField(String name, String type, String comment, RamlTypeV
 		// Add protected variable
 		JFieldVar field = this.pojo.field(JMod.PROTECTED, resolvedType, toJavaName(name), jExpression);
 
+		field.annotate(JsonProperty.class).param("value", typeDeclaration.name());
```

Comment:
Can we introduce check if `typeDeclaration.name()` differs from `toJavaName(name)` so we don't add `@JsonProperty` if it is not needed?
I would like to keep noise minimal in generated code..
Thanks!

Reply:
Agree, have fixed.
Thanks!